src/fm/cp_context_types.F

```diff
@@ -15,43 +15,65 @@
 !>      12.2003 created [Joost]
 !> \author Joost VandeVondele
 ! **************************************************************************************************
-MODULE cp_blacs_calls
+MODULE cp_context_types
```

Comment:
I'd prefer if we could keep `blacs`  in the name. Just `context` is too generic.

How about `cp_blacs_context_type` or simply `cp_blacs_type` ?

Reply:
I will change the name.